src/shared/constants/url.ts

```diff
@@ -1,9 +1,18 @@
 export enum AppRoutes {
   Home = "Home",
-  NotFound = "NotFound",
+  Any = "Any",
 }
 
 export const RoutePath: Record<AppRoutes, string> = {
   [AppRoutes.Home]: "/",
-  [AppRoutes.NotFound]: "*",
+  [AppRoutes.Any]: "*",
 };
+
+export const API_URL = "http://localhost:3000";
```

Comment:
Config the axios instead in MSW level do with that prefix

Reply:
how?

Reply:
Fixed